test/beforeRequest.test.js

```diff
@@ -146,9 +146,6 @@ beforeEach(() => {
     setMockFunctions();
     setTimeSinceLastResp(Date.now());
     setConfig(1); // set the CF config
-    // Mock the active commitments because XHR is not available
```

Comment:
Is removing a comment intendent?

Reply:
Yeah, this was an old leftover from when commitments were stored directly in the extension.